tests/webTables.spec.ts

```diff
@@ -0,0 +1,202 @@
+  });
+});
+
+test.describe("Web Tables-Veterinarians", async () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto("/");
+    await page.getByText("Veterinarians").click();
+    await page.getByText("All").click();
+  });
+
+  test("Validate specialty update", async ({ page }) => {
+    const targetRow = page.getByRole("row", { name: "Rafael Ortega" });
+    await expect(targetRow.locator("td").nth(1)).toHaveText("surgery");
+
+    await page.getByRole("link", { name: "Specialties" }).click();
+
+    await expect(page.locator("h2")).toHaveText("Specialties");
+
+    await page
+      .getByRole("row", { name: "surgery" })
+      .getByRole("button", { name: "Edit" })
+      .click();
+
+    await expect(page.locator("h2")).toHaveText("Edit Specialty");
+
+    const specialtyNameInputField = page.getByRole("textbox");
+    await specialtyNameInputField.click();
+    await specialtyNameInputField.clear();
+    await specialtyNameInputField.fill("dermatology");
+    await page.getByRole("button", { name: "Update" }).click();
+
+    expect(page.getByRole("row", { name: "dermatology" }));
+
+    await page.getByText("Veterinarians").click();
+    await page.getByText("All").click();
+    await expect(targetRow.locator("td").nth(1)).toHaveText("dermatology");
+
+    await page.getByRole("link", { name: "Specialties" }).click();
+    await page
+      .getByRole("row", { name: "dermatology" })
+      .getByRole("button", { name: "Edit" })
+      .click();
+    await specialtyNameInputField.click();
+    await specialtyNameInputField.clear();
+    await specialtyNameInputField.fill("surgery");
+    await page.getByRole("button", { name: "Update" }).click();
+  });
+});
+
+test.describe("Web Tables-Specialties", async () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto("/");
+    await page.getByText("Specialties").click();
+  });
+
+  test("Validate specialty lists", async ({ page }) => {
+    await page.getByRole("button", { name: "Add" }).click();
+    await page.locator("#name").fill("oncology");
+    await page.getByRole("button", { name: "Save" }).click();
+
+    await page.waitForTimeout(1000);
+
+    const specialties = [];
+    const specialtyRows = page.locator("tbody tr input");
+
+    for (let specialty of await specialtyRows.all()) {
+      const specialtyName = await specialty.inputValue();
+      specialties.push(specialtyName);
+    }
+
+    expect(specialties).toEqual([
+      "radiology",
+      "surgery",
+      "dentistry",
+      "New specialty",
+      "oncology",
+    ]);
+
+    await page.getByText("Veterinarians").click();
+    await page.getByText("All").click();
+
+    const targetRow = page.getByRole("row", { name: "Sharon Jenkins" });
+    await targetRow.getByRole("button", { name: "Edit Vet" }).click();
+
+    await page.locator(".dropdown-arrow").click();
+
+    const allBoxes = await page
+      .locator('//input[@type="checkbox"]/following-sibling::label')
+      .allInnerTexts();
```

Comment:
It can be as simple as: `page.locator(".dropdown-content div").allTextContents()`